File: testers/avro/java-vanilla/generated-and-checked-in/com/example/events/consumer/DynamicValueHandler.java
package com.example.events.consumer;

import com.example.events.DynamicValue;
import com.example.events.header.StandardHeaders;

/** Handler interface for dynamic-value topic events */
public interface DynamicValueHandler {
  /** Handle a message from the topic */
  void handle(String key, DynamicValue value, StandardHeaders headers);
}

File: testers/avro/kotlin/generated-and-checked-in/com/example/events/consumer/DynamicValueHandler.kt
package com.example.events.consumer

import com.example.events.DynamicValue
import com.example.events.header.StandardHeaders

/** Handler interface for dynamic-value topic events */
interface DynamicValueHandler {
  /** Handle a message from the topic */
  abstract fun handle(
    key: kotlin.String,
    value: DynamicValue,
    headers: StandardHeaders
  )
}